examples/demo_apollo.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020–2022 Cirq on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Demonstrates importing a quantum circuit from an OpenQASM 2.0 file, transforming it into
+the Apollo native gateset and connectivity, and then executing it on a simulator.
+"""
+import cirq
+from demo_common import demo
+
+from cirq_iqm.devices import Apollo
+from cirq_iqm.extended_qasm_parser import circuit_from_qasm
+
+
+def demo_apollo(do_measure=False, use_qsim=False):
+    """Run the demo using the Apollo architecture."""
+
+    print('\nApollo demo\n===========\n')
+    device = Apollo()
+    qasm_program = """
+        OPENQASM 2.0;
+        include "qelib1.inc";
+        qreg q[6];
+        creg meas[3];
+        U(0.2, 0.5, 1.7) q[1];
+        h q[0];
+        h q[2];
+        h q[4];
+        h q[5];
+        cx q[2], q[1];
+        cx q[3], q[5];
```

Comment:
Are these qubits coupled?

Reply:
No, and neither are the ones on line 51 (`circuit.insert(len(circuit) - 1, cirq.CXPowGate(exponent=0.723)(q5, q0))`). Fixed in next commit, thanks!